csp/impl/types/tstype.py

```diff
@@ -25,6 +26,45 @@ def __class_getitem__(cls, params):
 ts = TsType
 
 
+# Add core schema to TsType
+def __get_pydantic_core_schema__(cls, source_type, handler):
```

Comment:
similar Q, can we just add to class def as @classmethod

Reply:
Fixed